feature/policy_forwarding/otg_tests/ip_guev1_static_decap_subnet_range/README.md

```diff
@@ -0,0 +1,260 @@
+    *  Configure static routes for destination IPV4-DST2 and IPV6-DST2 towards ATE Port 2
+    *  Configure the UDP decap destination port as 6081 
+
+* DECAP-DST-SUBNET/30 subnet comprises of below IP prefixes:
+   - DECAP-DST_1/32
+   - DECAP-DST_2/32
+   - DECAP-DST_3/32
+   - DECAP-DST_4/32
+
+* DUT hosted IPv4 prefixes:
+  * Loopback Address = DECAP-DST_3/32
+
+### Test Procedure
+Traffic: 
+-  Generate the applicable GUE-encapsulated flow from ATE Port 1
+     -  Sent 1000 packets at the 10%  of the line rate.
+     -  Keep the frame size Internet Mix.
+   
+  
+### PF-1.4.1: GUE Decapsulation of inner IPv4 traffic over DECAP subnet range
+-  Push DUT configuration
+-  Initiate traffic flow type#1
+Verification:
+-  The outer header destination IP of the traffic falls within the configured decap range (DECAP-DST-SUBNET/30)
+-  The outer header destination port of the traffic (UDP 6081) matches the configured non-default UDP decap port criteria
+-  The DUT will decapsulate the outer header and perform the lookup based on the inner IP address
+-  The TTL bits will not be copied to the inner header
+-  The inner header TTL value will be decremented by 1
+-  The inner packet's DSCP value (32) will be preserved
+-  The DUT will forward the traffic towards ATE Port 2
+-  The relevant DUT counters will reflect 1,000,000 decapsulated packets
+-  ATE Port 2 receives 1000 packets structured as RX-Flow-type#2
+-  No packet loss should be observed
+
+### PF-1.4.2: GUE Decapsulation of inner IPv6 traffic over DECAP subnet range
+-  Push DUT configuration.
+-  Initiate traffic flow type#2
+Verification:
+-  The outer header destination IP of the traffic falls within the configured decap range (DECAP-DST-SUBNET/30)
+-  The outer header destination port of the traffic (UDP 6081) matches the configured non-default UDP decap port criteria
+-  The DUT will decapsulate the outer header and perform the lookup based on the inner IP address
+-  The TTL bits will not be copied to the inner header
+-  The inner header TTL value will be decremented by 1
+-  The inner packet's DSCP value (32) will be preserved
+-  The DUT will forward the traffic towards ATE Port 2
+-  The relevant DUT counters will reflect 1,000,000 decapsulated packets
+-  ATE Port 2 receives 1000 packets structured as RX-Flow-type#4
+-  No packet loss should be observed
+
+### PF-1.4.3: GUE Decapsulation of inner IPv4 traffic using non-default and unconfigured GUE UDP port (Negative).
+-  Push DUT configuration.
+-  Initiate traffic flow type#3
+Verification: 
+-  The outer header destination IP of the traffic falls within the DECAP-DST-SUBNET/30 range
+-  The outer header destination UDP port (6085) of the traffic is not configured for decapsulation, therefore it does not match the decapsulation criteria.
+-  The DUT should not decapsulate these packets. Packets should be dropped since no specific drop rule exists for unmatched GUE
```

Comment:
I think the packets will be dropped due to no-route.  ie: since the policy-forwarding rule doesn't match the packet, then the next-step is for the DUT to lookup the destination.  Since that is the loopback address, the packet will be sent to the device controller/CPU.  What should happen in this case?